.storybook/config.js

```diff
@@ -20,5 +21,6 @@ addDecorator((story) => (
 ));
 addDecorator((story) => <ThemeProvider theme={theme}>{story()}</ThemeProvider>);
 addDecorator(withA11y);
+addDecorator(withKnobs);
```

Comment:
Moved this to a global decorator as storybook started throwing errors with the "local" ones (or at least the one in the Accordion stories).